tests/page_tests.py

```diff
@@ -236,14 +243,17 @@ def get_url_basename(url):
             else:
                 return url
 
-        def check_link(lnk, sess, filenames, folders):
-
-            if not check_skip_conditions(lnk, filenames, folders):
-                failure = try_to_connect(lnk, sess)
+        def check_link(link, session, filenames, folders):
+            if not check_skip_conditions(link, filenames, folders):
+                if get_url_basename(link) == "github.com" and any([f"{wiki.name}/wiki" in link for wiki in WIKI_INCLUDELIST]):
+                    # Get the wiki page name and append .md to it to check if the file exists in the local wiki repo
+                    _page_file_name = f"{link.split('/')[-1]}.md"
+                    return check_if_link_to_wiki_page(_page_file_name, filenames, folders)
+                failure = try_to_connect(link, session)
```

Comment:
I think the code would look cleaner and would make the logic clearer if you put the connection failure check within an else statement. It makes it more clear that the algorithm is to check if the link is a full link to one of our wiki pages, and if not it is an external github link and we want to check if we can connect to it.